backend/app/api/endpoints/knowledge.py

```
@@ -535,6 +535,27 @@ def _index_document_background(
                 logger.info(
                     f"Updated document {document_id} is_active to True and status to enabled after successful indexing"
                 )
+
+                # Trigger document summary generation automatically after indexing
+                try:
+                    summary_enabled = getattr(settings, "SUMMARY_ENABLED", True)
```

Ruff (0.14.10)

Undefined name `settings`

(F821)